src/main/java/com/abcdedu_backend/post/service/PostService.java

```diff
@@ -25,57 +32,96 @@ public class PostService {
     private final PostReposiroty postReposiroty;
     private final BoardService boardService;
     private final MemberService memberService;
+    private final FileHandler localFileHandler;
+
```

Comment:
p2: 이거 fileHandler로 변경해주시면 좋을 것같아요. FileHandler 인터페이스로 뺀 거기 때문에 변수명이지만 의존할 필요는 없다고 봅니다.
```suggestion
    private final FileHandler fileHandler;

```

Reply:
이 부분 빈 주입할 때 같은 타입 빈이 여러개면 어떤걸 넣을지 모호해져서 이름으로 표현한 것인데,, Config를 따로 만들거나 우선순위 정하는 애노테이션을 handler에 붙일 예정이라면 저렇게 바꾸겠습니다!

Reply:
우선 저도 Config나 hadler에 애노테이션 다는 방법이 더 나을 것 같아서 일단 말씀하신대로 바꿀게요!

Reply:
LocalFileHandler class에 @Primary 달아놨습니다! 이거 없으면 에러나니 참고 부탁드려용 ㅎㅎ

Reply:
아 그렇게 해도 좋겠네요! 감사합니다